core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -1602,8 +1602,20 @@
   // default value is 2 days
   public static final String CARBON_SEGMENT_LOCK_FILES_PRESERVE_HOURS_DEFAULT = "48";
 
+  /**
+   * The number of invisible segment info which will be preserved in tablestatus file,
+   * if it exceeds this value, they will be removed and write to tablestatus.history file.
+   */
+  @CarbonProperty
+  public static final String CARBON_INVISIBLE_SEGMENTS_PRESERVE_COUNT =
+      "carbon.invisible.segments.preserve.count";
+
+  /**
+   * default value is 20, it means that it will preserve 20 invisible segment info
```

Comment:
The default is 200 right. Please update comment

Reply:
Done